lab_1/main.py

```diff
@@ -0,0 +1,125 @@
+"""
+Lab 1
+A concordance extraction
+"""
+
+
+def tokenize(text: str) -> list:
+    """
+    Splits sentences into tokens, converts the tokens into lowercase, removes punctuation
+    :param text: the initial text
+    :return: a list of lowercase tokens without punctuation
```

Comment:
Typo: `lowercased`

Reply:
PyCharm gives typo in "lowercased" :) `Typo: In word 'lowercased'` Will be corrected